packages/components/psammead-media-player/src/Placeholder/index.jsx

```diff
@@ -28,6 +30,27 @@ const StyledPlayButton = styled(PlayButton)`
   /* stylelint-enable */
 `;
 
+const NoScriptHiddenStyle = (
+  // this would be rendered multiple times if we have more than one audio/video content on a page,
+  // and only the first one would be used leaving a lot oof unused styles.
+  // we have to consider moving it to the <head> in simorgh.
```

Comment:
something like [this](https://github.com/bbc/simorgh/blob/latest/src/server/Document/component.jsx#L50)